data/org.pantheon.desktop.gala.gschema.xml.in

```diff
@@ -163,11 +163,6 @@
 			<default>false</default>
 			<summary>Make parent windows that spawn a modal attached dialog being dimmed</summary>
 		</key>
-		<key type="d" name="alt-tab-window-opacity">
```

Comment:
Don't remove gsettings keys!
Add "<description>DEPRECATED: This key is deprecated and ignored.</description>" like it is done for other ones.